pkgs/build-support/fetchgithub/default.nix

```diff
@@ -2,15 +2,15 @@
 
 { owner, repo, rev, name ? "source"
 , fetchSubmodules ? false, leaveDotGit ? null
-, deepClone ? false, private ? false
+, deepClone ? false, private ? false, forceFetchGit ? false
 , githubBase ? "github.com", varPrefix ? null
 , ... # For hash agility
 }@args:
 let
   baseUrl = "https://${githubBase}/${owner}/${repo}";
   passthruAttrs = removeAttrs args [ "owner" "repo" "rev" "fetchSubmodules" "private" "githubBase" "varPrefix" ];
```

Comment:
Seems like `"forceFetchGit"`should be added to this list, because I currently get this with `forceFetchGit = true`:

```
error: anonymous function at /nix/store/81gc6d30wsc6a320vhpbdiw8ml2x2fcs-source/pkgs/build-support/fetchgit/default.nix:15:1 called with unexpected argument 'forceFetchGit'

       at /nix/store/81gc6d30wsc6a320vhpbdiw8ml2x2fcs-source/pkgs/build-support/fetchgithub/default.nix:39:1:

           38|
           39| fetcher fetcherArgs // { meta.homepage = baseUrl; inherit rev; }
             | ^
(use '--show-trace' to show detailed location information)
```

Reply:
Thanks! I created #145511 to fix that.